src/Microsoft.VisualBasic/tests/VB/TextFieldParserTests.vb

```diff
@@ -0,0 +1,180 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System.Runtime.CompilerServices
+Imports Microsoft.VisualBasic.FileIO
+Imports Xunit
+
+Namespace Microsoft.VisualBasic.Tests.VB
+    Public NotInheritable Class TextFieldParserTests
+        Inherits FileIOTestBase
+
+        Private Const HeaderLine As String = "seq,oneword,age,street,city,state,zip,dollar,pick,date,paragraph"
+
+        Private Const CSVData As String =
+                    HeaderLine & vbCrLf &
+            "1,Test,99,Two Words,Rumnoman,KS,80586,$8333.85,GREEN,09/28/1929,""Ipmadaw udoteac vu ote ozehop ane lujat, lar buh cugi lef owici tat liz cogde.
+Fatigiug kojitfu taapi iz alujok me zimipa koz latkekij vem fo si cepzizhub li por ejcirlu.
+Ewujajlu mivec tooju ad bu cowic irtafit ehaoca vojhehfo aztidun zo wezecmo abe muz wikhutwen idce.
+Vesu sejawga tef lahi dirueg si uwmac bidiw nowidza daime sapmim ki casdun urokir tawdac rahaw beiweed."""
```

Comment:
Please move this constant to the one test where it is used.

Reply:
Please limit text to actual words rather than arbitrary strings of letters that may have meaning in other languages.

Reply:
Please test:
Empty fields: "...,,"""",..."
Leading and trailing spaces: "..., one,two ,..." 
Spaces only: "..., ," & vbCrLf & ",..."

Reply:
Constants moved
The text is Ipsum Lorum, which is what I found other test this like used.
Added other tests